### PR TITLE
Add insertion sort

### PR DESCRIPTION
**Pull request by Sunbelife**
Copy the brute force sort header, leave Suu to check.

See issue #25

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Llama-Group/Project-Llama/pull/44" target="_blank">ReviewNinja

File: llama_cpp/src/Sort/Sort.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include <string>
 #include <iostream>
 #include <stdexcept>
-
+//Sunbelife
```
Comment (0xsuu) [inline llama_cpp/src/Sort/Sort.h]: Remove this line

**Comment by 0xsuu** [comment]
@Sunbelife Well done, now implement your insertion sort and add an example to test your insertion sort.

**Comment by 0xsuu** [comment]
@Sunbelife Build problem solved, can type 

```
-e sort_test
```

now.

File: llama_cpp/example/sort_test/sort_test.cpp [hunk 2]
```diff
@@ -1,14 +1,17 @@
 
+#include "Sort/InsertionSort.h"
 #include "Sort/BruteForceSort.h"
 
 class CustomisedClass {
 public:
 	CustomisedClass() {}
-	CustomisedClass(int input) {
+	CustomisedClass(int input) 
+	{
```
Comment (0xsuu) [inline llama_cpp/example/sort_test/sort_test.cpp]: Change back to previous bracket style.

File: llama_cpp/example/sort_test/sort_test.cpp [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 		this->input = input*10;
 	}
 	
-	int getInput() const {
+	int getInput() const 
+	{
```
Comment (0xsuu) [inline llama_cpp/example/sort_test/sort_test.cpp]: Change back to previous bracket style.

**Comment by 0xsuu** [comment]
@Sunbelife Pull the newest master branch and merge, that added coverage test.

**Comment by 0xsuu** [comment]
OK

**Comment by 0xsuu** [comment]
@Sunbelife now i want you to add a new branch to add benchmark test of your sort.

**Pull request merged**